unittests/ThunkLibs/generator.cpp

```diff
@@ -415,9 +415,17 @@ TEST_CASE_METHOD(Fixture, "UnknownAnnotation") {
 TEST_CASE_METHOD(Fixture, "VersionedLibrary") {
   const auto output = run_thunkgen_host("", "template<auto> struct fex_gen_config { int version = 123; };\n");
 
+#if CLANG_VERSION_MAJOR >= 19
+#define ENDSWITH ends_with
+#else
+#define ENDSWITH endswith
+#endif
+
   CHECK_THAT(output, matches(callExpr(callee(functionDecl(hasName("dlopen"))), hasArgument(0, stringLiteral().bind("libname"))))
                        .check_binding(
-                         "libname", +[](const clang::StringLiteral* lit) { return lit->getString().endswith(".so.123"); }));
+                         "libname", +[](const clang::StringLiteral* lit) { return lit->getString().ENDSWITH(".so.123"); }));
```

Comment:
Could you just wrap the line itself in the ifdef, or disable the VersionedLibrary test entirely? Using a macro to expand to a member function name seems unorthodox and can easily be avoided that way.